core/templates/dev/head/pages/state_editor/state_content_editor_directive.html

```diff
@@ -37,4 +50,42 @@
     <button type="button" class="btn btn-default pull-right" ng-click="cancelEdit()">Cancel</button>
     <div style="clear: both;"></div>
   </div>
+  <div ng-if="isCardHeightLimitReached && showCardHeightLimitWarning">
+    <span class="oppia-card-height-limit-warning">
+      <i class="material-icons" uib-tooltip="Hide warning" ng-click="hideCardHeightLimitWarning()">&#xE88F;</i>
+      <span translate="I18N_CARD_HEIGHT_LIMIT_WARNING_MESSAGE"></span>
+    </span>
+  </div>
 </div>
+
+<style>
+  .oppia-card-height-limit-warning {
+      color: #0277BD;
+      padding-top: 2px;
+  }
```

Comment:
Add a new line after each style. (Likewise line 84 in this file.)

Reply:
Done.